### ISSUE TITLE
{Linked:Bug9781003;Linked:Bug9768640;Freeway-Bug} [Adaptive Cards> Designer]: Screen reader not announcing any information after clicking on 'Show next/previous' buttons in date picker.

### ISSUE THREAD

**Issue by vagpt** [microsoft/AdaptiveCards#8097]
### Target Platforms

Other

### SDK Version

1.5

### Application Name

Adaptive Cards

### Problem Description

**Test Environment:**
OS Version: 22H2 (Build 25247.1000)
Browser: Edge Dev (109.0.1495.2)

Screen Reader: NVDA

**Repro Steps:**
1. Open the above URL in edge dev browser.
2. Navigate to the 'Designer' tab through tab key and activate it.
3. Navigate to the 'Sample data Editor' and ' Card payload editor' and attach the provided JSON file.
4. Navigate to 'Preview mode' and activate it.
5. Navigate to 'Enter a Date' button and activate it.
6. Navigate to the 'Show Next month' button and press enter key and observe the issue. 

**Actual Result:**
Screen reader not announcing any information after clicking on 'Show next/previous' buttons in date picker.

**Expected Result:**
Screen reader has to announce information after clicking on 'Show next/previous' buttons in date picker.

**User Impact:**
It will impact the screen reader user if the screen reader does not convey the information while activating any control on the screen as user won't be able to know whether control is activated or not?

**MAS Reference:**
https://liquid.microsoft.com/Web/Object/Read/MS.Accessibility/Requirements/01.03.01

"Have feedback to share on Bugs? Please help fill [Trusted Tester Bug Feedback (office.com)](https://nam06.safelinks.protection.outlook.com/?url=https://forms.office.com/pages/responsepage.aspx?id%3Dv4j5cvGGr0GRqy180BHbR5dUqGQgJ9lItv-jqY8F09lURVdYTENIRkY2UzlCR0NKV1E2VUE5OVpDVy4u&data=04%7c01%7cv-apoorva%40microsoft.com%7c551850210d444be9b28308d9691e3dde%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c637656400998658942%7cUnknown%7cTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7c1000&sdata=l6w0SvVcMSvyGnJ6%2B6uSSgd3braF2dZ7/AdBrqqzIR8%3D&reserved=0) “.

### Screenshots

https://user-images.githubusercontent.com/95913748/205279210-fa4e5733-2655-4382-a5d7-6683d3634fa5.mp4

### Card JSON

```JSON
{
  "$schema": "http://adaptivecards.io/schemas/adaptive-card.json",
  "type": "AdaptiveCard",
  "version": "1.3",
  "body": [
    {
      "type": "TextBlock",
      "text": "Date Input"
    },
    {
      "type": "Input.Date",
      "id": "date",
      "placeholder": "Enter a date",
      "value": "2017-10-12"
    }
  ],
  "actions": [
    {
      "type": "Action.Submit",
      "title": "OK"
    }
  ]
}
```

**Comment by anna-dingler** [comment]
Hi @vagpt ,

Our team uses HTML's `<input type="date">` for this control. Because of this, we cannot make any modifications to the contents of the Calendar picker or Time picker.

This appears to be an issue with NVDA. I tested the base input control with the following HTML on both Chrome and Edge, and the behavior was the same as seen on our website.

**Test HTML**
```HTML
<input type="date">
```

Please let me know if you have any questions or if there is anything else we can do to help resolve this issue. Thanks!

**Comment by vagpt** [comment]
Hi @anna-dingler,

We have a comment from the dependent team i.e.
Comment -> "@Satish Pithani Added the attachment using Narrator as a screen reader.
PFA: "9781003-Narrator screen reader announcing next and previous month information incorrectly.mp4".
Observation:
Narrator announcing incorrect information (month that is changed) after clicking on show next and show previous month buttons."

So could you please look into this issue again as this issue is repro with Narrator screen reader as well and help us.

**Attachment:**

https://user-images.githubusercontent.com/95913748/222360815-e48bbf9e-476b-4258-a681-39ed96edcb69.mp4

**Dependent Bug->**
https://dev.azure.com/MicrosoftIT/OneITVSO/_workitems/edit/9781003

**Comment by anna-dingler** [comment]
Hi @vagpt,

Good catch that the issue repros with Narrator as well.

I checked the test HTML with Narrator and saw the same behavior.

## Test HTML
```<input type="date">```

## Behavior

https://user-images.githubusercontent.com/98650930/222589318-8065550b-2918-4c36-8509-53d8acf783d2.mp4

For reference, this is what I see in DevTools:
<img width="401" alt="image" src="https://user-images.githubusercontent.com/98650930/222590412-61dfddf2-cd46-402b-84e8-0e6e426af7c4.png">

Since the issue repros on both screen readers for a simple base case, this could be a bug on Edge.

Would you be able to follow up with the Edge team?

**Comment by vagpt** [comment]
Thanks @anna-dingler for the updates.

Sure, I will share this bug to Edge team and update the bug id here.

**Comment by vagpt** [comment]
Hi @anna-dingler 

 Now this issue can be tracked using the below URL as it is logged under EDGE VSO.

https://microsoft.visualstudio.com/Edge/_workitems/edit/43722704
